pdsc/ingest.py

```diff
@@ -218,8 +235,14 @@ def store_segment_tree(outputfile, segments):
 
     :param segments:
         a collection of :py:class:`~pdsc.segment.TriSegment` objects
+
+    :body_radius:
+        celestial body radius, default is Mars
     """
-    tree = SegmentTree(segments)
+    if body_radius == MARS_RADIUS_M:
+        tree = SegmentTree(segments)
+    else:
+        tree = SegmentTree(segments, True, body_radius)
```

Comment:
Can remove the `if`/`else` block and just always pass in `body_radius=body_radius` argument.